main.c

```diff
@@ -238,6 +247,19 @@ enum option_indices {
         *t++ = 0;\
     } while (false)
 #endif
+
+struct preset {const char* name; const char* settings; int exp; int money;};
```

Comment:
we could add `--presets` and `--preset <name>` to the CLI version to match features

Reply:
if be we, you mean, you, then I agree :P. I don't think many people use the CLI version and this is just QoL-Feature so I would not see this as a high priority.

Reply:
if presets are not supported by the CLI version, then `--presets.json` should not be compiled in:
* remove from help
* `#ifdef` away in the option parsing (so you get an error specifying it)
* `#ifdef` away the implementation (so you don't get unused function warning)